Overview.src.html

```diff
@@ -3210,6 +3323,73 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
      <li>If <var>cookies</var> is not the empty string, append
      `<code title=http-cookies>Cookie</code>`/<var>cookies</var> to <var>httpRequest</var>'s
      <span title=concept-request-header-list>header list</span>.
+
+     <li><p>If the user agent supports <span title=concept-token-binding>Token Binding</span>,
+       build a <code title="">Sec-Token-Binding</code> <span title=concept-header>header</span>
+       by running these substeps:
+       <p class="note no-backref">A <code title="">Sec-Token-Binding</code>
+       <span title=concept-header>header</span> allows a server
+       to bind cookies that it issues to a <span title=concept-token-binding-id>Token Binding Id</span>,
+       and to verify that cookies presented by the user agent are bound to the correct
+       <span title=concept-token-binding-id>Token Binding Id</span>.
+
+       <ol>
+         <li><p>Set the <var>httpRequest</var>'s
+           <span title=concept-request-use-token-binding>use-token-binding flag</span>.
+         <li><p>Initialize <var>tokenBindingMessage</var> to null.
+           <li><p>Let <var>providedTokenBindingId</var> be the result of looking up the
+          <span title=concept-token-binding-id>Token Binding Id</span> for the <var>httpRequest</var>'s
+          <span title=concept-request-current-url>current url</span>.
+
+          <p class="note no-backref">A user agent negotiates the use of Token Binding with a server
+          when it sets up a TLS connection to that server, creating a
+          <span title=concept-token-binding-id>Token Binding Id</span> at the end
+          of the negotiation. The <span title=concept-token-binding-id>Token Binding Id</span>
+          is saved and can then simply be looked up whenever needed.
+          If the user agent is unable to negotiate Token Binding with a server, it will not create
+          a <span title=concept-token-binding-id>Token Binding Id</span>.
+          If the <span title=concept-token-binding-id>Token Binding Id</span> for a server is
+          deleted by a user agent (e.g., as a result of some user action), the user agent will
+          need to re-establish a TLS connection to negotiate Token Binding and create a new
+          <span title=concept-token-binding-id>Token Binding Id</span>.
+
+          <li><p>If <var>providedTokenBindingId</var> is not null, then add a
+            <code title="">provided_token_binding</code> to <var>tokenBindingMessage</var>.
```

Comment:
providedTokenBindingId is used to generate a provided_token_binding per the TOKBIND-PROTOCOL spec.